src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -207,19 +189,15 @@ public double getTotalDebt() {
     public double getTotalCredit() {
 
         double result = 0.0;
-        for (Person x : personList) {
-            if (x.getMoneyOwed().getAmount() < 0) {
-                result += -x.getMoneyOwed().getAmount();
+        for (Person person : personList) {
+            if (person.getMoneyOwed().getAmount() > 0) {
+                result += person.getMoneyOwed().getAmount();
```

Comment:
Again, similar to my comment above, HomeCard shouldn't have to know what amount > 0 means. Do you think replacing with `!person.getMoneyOwed().userOwesMoney()` would better follow the "tell don't ask principle"?